doc/scala.txt

```diff
@@ -1,9 +1,46 @@
 *scala.txt* Syntax highlighting and helper functions for the Scala language.
 
+This plugin is only available if 'compatible' is not set.
+{Vi does not have any of this}
+
+==============================================================================
 INTRODUCTION                                    *scala*
 
-Syntax highlighting and helper functions for the scala language.
+Syntax highlighting and helper functions for the scala language. Extras
+include:
+
+  - Sorting of import statements, configurable to your conventions.
+  - Tagbar support to navigate definitions within a file in the plugin's
+    sidebar window.
+  - ...and probably more that we've forgotten to update in this doc.
+
+==============================================================================
+OPTIONS                                         *scala-options*
+
+Use these options to control behavior of the plugin. Default values are
+indicated in the examples.
 
+                                           *'g:scala_use_builtin_tagbar_defs'*
+If you are using the Tagbar Vim plugin, vim-scala includes a Tagbar type
+definition and ctags definition for Scala, so you can use Tagbar immediately.
+If you have your own ctags definition in `~/.ctags` and prefer to use it, set
+this option to 0 (we would appreciate contributions if you've improved the
+ctags definition!).
+
+Note that Tagbar's ctags definition for Scala is not used to generate a
+|tags| file that Vim can use to navigate to definitions in other files, only
+for the plugin sidebar. Feel free to copy `vim-scala/ctags/scala.ctags` into
```

Comment:
Yeah, Tagbar never uses tags files on disk, it just fires a `ctags` command individually for every file when Tagbar is opened and after buffer updates. That's cool for people that just want to see the structure of the current file and haven't yet come to the realization that all that messing around with file generation really is worth it...